### ISSUE TITLE
Retain original filename capitalization

### ISSUE THREAD

**Issue by lencioni** [tdooner/mosaic#18]
I'm seeing all lowercase filenames in mosaic, but filenames with capital letters in dropbox. Not sure if these are the same files, but this seems like a bug.

**Comment by tdooner** [comment]
Yeah, mosaic downcases the path for easy matching. But it could retain the original path for easy human consumption.

**Comment by lencioni** [comment]
What do you mean by easy matching? I just find it confusing.

**Comment by tdooner** [comment]
"easy matching" on the backend, so that if files' names change capitalization then they are still considered to be the same file between successive dropbox sync attempts.
